x-pack/docs/en/rest-api/security/enroll-kibana.asciidoc

```diff
@@ -17,6 +19,10 @@ Enables a {kib} instance to configure itself for communication with a secured {e
 Use this API to enable a {kib} instance to configure itself for communications
 with an {es} cluster that already has security features enabled.
 
+When you first start {es}, the installation process generates an enrollment token for
+{kib}. You enter this enrollment token when starting {kib} for the first time, and
+{kib} calls the enroll {kib} API to verify the fingerprint of the enrollment token.
```

Comment:
if we keep this section, I'll get back with a suggestion to replace "verify the fingerprint of the enrollment token" as this is not accurate

Reply:
I think that we can remove this section. As you indicated in a previous comment, end users probably don't need to know the internal workings (I agree!). Instead, I propose that we update the previous sentence to indicate that only Kibana uses this API. For example:
>Kibana uses this API internally to configure itself for communications with an {es} cluster that already has security features enabled.